File: backend/src/main/java/com/desarrollo/raffy/dto/ReviewFromBackToFrontDTO.java
package com.desarrollo.raffy.dto;

import com.desarrollo.raffy.model.DeliveryStatus;

import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Data

@Getter @Setter
@NoArgsConstructor

public class ReviewFromBackToFrontDTO {
    private Long eventId;
    private String eventTitle;
    private String name;
    private String surname;
    private String nickname;
    private Double score;
    private DeliveryStatus delivery;
    private String comment;
}

File: frontend/src/app/models/review/reviewFromBacktoFrontDTO.ts
import { AwardAlignment } from "./AwardAlignment";
import { CommunicationRating } from "./CommunicationRating";
import { DeliveryStatus } from "./DeliveryStatus";

export interface reviewFromBacktoFrontDTO {
    name: string,
    surname: string,
    nickname: string|null,
    eventTitle: string,
    eventId: string,
    delivery: DeliveryStatus,
    awardAlingment: AwardAlignment,
    communicationRating: CommunicationRating,
    score: number,
    comment: string
}